ci/set_bindings_version.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+# source "${WORKDIR}/ci/export_lib_versions.sh"
+
+declare -A LIB_NAME_TO_PACKAGE=(
+    [libtelio]=github.com/NordSecurity/libtelio-go
+    [libdrop]=github.com/NordSecurity/libdrop-go
+)
+
+declare -A LIB_NAME_TO_VERSION
+[[ -n "${LIBTELIO_VERSION:-}" ]] && LIB_NAME_TO_VERSION[libtelio]=$LIBTELIO_VERSION
+[[ -n "${LIBDROP_VERSION:-}" ]] && LIB_NAME_TO_VERSION[libdrop]=$LIBDROP_VERSION
+
+declare -A LIB_NAME_TO_TAG_VERSION
+[[ -n "${LIBTELIO_TAG_VERSION:-}" ]] && LIB_NAME_TO_TAG_VERSION[libtelio]=$LIBTELIO_TAG_VERSION
+[[ -n "${LIBDROP_TAG_VERSION:-}" ]] && LIB_NAME_TO_TAG_VERSION[libdrop]=$LIBDROP_TAG_VERSION
+
+lib_name=$1
```

Comment:
Maybe check if param exists or add an error message when `lib_name` not found. Or both? :smile:

Reply:
Should already fail thanks to `set -euxo pipefail` I think.